src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -68,9 +72,17 @@
  * <p>
  * app-dev.properties //separator is '-'
  * </p>
+ *
+ * <p><b>checkedLanguageCodes</b> which allows to specify language codes of
+ * translation files which must exist in project. For example, if you need
+ * to check that Japanese and French localization files are present in your project,
+ * you should set checkedLanguageCodes to "ja, fr".
```

Comment:
If you'll set it like here, will the check also accept other translations? Please clarify in docs.
Also, what about ROOT translation (no code in file name)?